Source/sync.cpp

```diff
@@ -312,7 +312,7 @@ uint32_t OnSyncData(const TCmd *pCmd, int pnum)
 				continue;
 
 			if (GetLevelForMultiplayer(*MyPlayer) == level) {
-				SyncMonster(pnum, monsterSyncs[i]);
+				SyncMonster(pnum > MyPlayerId, monsterSyncs[i]);

```

Comment:
Not directly related to this cleanup-pr:
What is the background of this comparison? I don't know any other part of the code that does this.

Reply:
It's planned as one of the next things for one I resume the object cleanups :)

https://github.com/diasurgical/devilutionX/pull/4571#discussion_r872933663